xarray_sentinel/sentinel1.py

```diff
@@ -223,13 +226,23 @@ def open_burst_dataset(
     swath_timing = esa_safe.parse_swath_timing(annotation_path)
     linesPerBurst = swath_timing["linesPerBurst"]
     samplesPerBurst = swath_timing["samplesPerBurst"]
+    first_azimuth_time = pd.to_datetime(
+        swath_timing["burstList"][burst_position]["azimuthTime"]
+    )
+    azimuth_time_interval = pd.to_timedelta(
```

Comment:
Or, eliminate `pandas` here is a bit more involved: `np.timedelta64(image_information["azimuthTimeInterval"] * 1_000_000, "ms")`